airflow/jobs/scheduler_job_runner.py

```diff
@@ -1675,7 +1675,7 @@ def check_trigger_timeouts(self, session: Session = NEW_SESSION) -> None:
             .values(
                 state=TaskInstanceState.SCHEDULED,
                 next_method="__fail__",
-                next_kwargs={"error": "Trigger/execution timeout"},
+                next_kwargs={"error": "Trigger timeout"},
```

Comment:
this i change just to be clear the scenario is exclusively trigger timeout, not execution_timeout, IIUC